doc/topics/jinja/index.rst

```diff
@@ -1473,6 +1476,15 @@ Returns:
 File filters
 ------------
 
+.. jinja_ref:: connection_check
+
+``connection_check``
+-------------
+
+.. versionadded:: Fluorine
+
+Return the IP resolved by DNS. This is an alias of `dns_check`.
```

Comment:
The `dns_check` needs double backtcks instead of single backticks in order to display that text like you're intending. RST syntax like 2 ticks, whereas markdown can use one or two.